server/src/business-layer/services/StripeService.test.ts

```diff
@@ -1,5 +1,6 @@
 import StripeService from "./StripeService"
 import { productMock } from "test-config/mocks/Stripe.mock"
+import Stripe from "stripe"
```

Comment:
Nit: Don't think we need to import this?

Reply:
I was getting an error because the type of `param` wasn't getting detected in line 28 hence, I had to explicitly mention it's an object type of `Stripe.ProductUpdateParams`

Reply:
Ok makes sense